pkg/pillar/cmd/vaultmgr/vaultmgr.go

```diff
@@ -512,13 +512,21 @@ func publishVaultKey(ctx *vaultMgrContext, vaultName string) error {
 		}
 		keyBytes, err := etpm.FetchSealedVaultKey(log)
 		if err != nil {
-			return fmt.Errorf("Failed to retrieve key from TPM %w", err)
+			return fmt.Errorf("failed to retrieve key from TPM %w", err)
 		}
 
 		encryptedKey, err := etpm.EncryptDecryptUsingTpm(keyBytes, true)
```

Comment:
Do we have a way to inject faults here so we can create automatic tests for this broken TPM handling logic?

Reply:
not at the moment, but I'm looking into it.